### PR TITLE
CI-33 Quicklook test compatible with JDK8/11

### PR DESCRIPTION
**Pull request by jGauravGupta**
# Description
This is a bug fix in Quicklook test to make it compatible with JDK 8 & 11.

# Important Info

### Related PR
https://github.com/payara/Payara/pull/3842/

# Testing

### Test suites executed
- Quicklook

### CODE REVIEW

**Comment by MarkWareham** [comment]
Jenkins test please

**Comment by MarkWareham** [comment]
Jenkins, test please

**Comment by jGauravGupta** [comment]
Jenkins test please

**Comment by jGauravGupta** [comment]
Jenkins test please

File: appserver/tests/quicklook/ejb/mdb/src/test/mdb/MDBTests.java [hunk 1]
```diff
@@ -84,10 +84,15 @@ public void deployJMSAppTest(String mdbAppDir) throws Exception {
...
-        cmd = APPCLIENT + " -targetserver" + " localhost:3700" + " -client " + cwd + File.separator + mdbAppDir + mdbApp + "Client.jar "
-                + "-name ejb-ejb30-hello-mdb-client ";
-        execReturn = RtExec.execute("MDBTests.runJMSAppTest", cmd);
+        String clientJar = cwd + File.separator + mdbAppDir + mdbApp + "Client.jar";
+        String gfClientJar = GLASSFISH_HOME + File.separator + "lib" + File.separator + "gf-client.jar";
+        cmd = APPCLIENT + " " + GLASSFISH_APPCLIENT_MAIN_CLASS_NAME
+                + " -client " + clientJar
+                + " -targetserver" + " localhost:3700"
+                + " -name ejb-ejb30-hello-mdb-client"
+                + " -cp " + gfClientJar + ";" + clientJar;
```
Comment (Pandrex247) [inline appserver/tests/quicklook/ejb/mdb/src/test/mdb/MDBTests.java]:
```suggestion
                + " -cp " + gfClientJar + File.pathSeparator + clientJar;
```

**Comment by jGauravGupta** [comment]
Jenkins test please

**Comment by MarkWareham** [comment]
Superseded by #4352

**Pull request closed without merging**